### PR TITLE
tools: add more sub commands for tikv-ctl

### PR DESCRIPTION
**Pull request by hicqu**
Signed-off-by: qupeng <qupeng@pingcap.com>

### CODE REVIEW

File: tools/tikv-control.md [hunk 1]
```diff
@@ -19,6 +19,12 @@ TiKV Control (`tikv-ctl`) is a command line tool of TiKV, used to manage the clu
     $ tikv-ctl --ca-path ca.pem --cert-path client.pem --key-path client-key.pem --host 127.0.0.1:21060 <subcommands>
     ```
 
+    However sometimes `tikv-ctl` comunicates with `PD` instead of `TiKV`, in which case we need to use `--pd` option instead of `--host`. Here is an example:
```
Comment (lilin90) [inline tools/tikv-control.md]: - Add a comma after "However"
- comunicates -> com**m**unicates
- we -> you
- use `--pd` option -> use the `--pd` option

File: tools/tikv-control.md [hunk 2]
```diff
@@ -99,6 +105,28 @@ In this command, the key is also the escaped form of raw key.
 
 To print the value of a key, use the `print` command.
 
+### Print some properties about region
```
Comment (lilin90) [inline tools/tikv-control.md]: region -> Region

File: tools/tikv-control.md [hunk 3]
```diff
@@ -99,6 +105,28 @@ In this command, the key is also the escaped form of raw key.
...
+
+In order to record region state details, `TiKV` writes some statistic into SST files of regions.
+For take a look at these properties, run `tikv-ctl` with sub-command `region-properties`:
```
Comment (lilin90) [inline tools/tikv-control.md]: (Put the two lines in a same line) -> In order to record **Region** state details, **TiKV** writes some **statistics** into **the** SST files of **Regions**. **To view** these properties, run `tikv-ctl` with sub-command `region-properties`:

File: tools/tikv-control.md [hunk 4]
```diff
@@ -99,6 +105,28 @@ In this command, the key is also the escaped form of raw key.
...
+mvcc.max_ts: 0
+mvcc.num_rows: 0
+mvcc.num_puts: 0
+mvcc.num_versions: 0
+mvcc.max_row_versions: 0
+middle_key_by_approximate_size:
+```
+
+The properties can be used for tell the region is health or not. If not, then we can use them to fix the region.
+E.g. split the region manually by the `middle_key_approximate_size`.
```
Comment (lilin90) [inline tools/tikv-control.md]: Update these two lines to:

The properties can be used **to check whether the Region is healthy** or not. If not, **you can** use them to fix the **Region**. For example, splitting the **Region** manually by ~~the~~ `middle_key_approximate_size`.

File: tools/tikv-control.md [hunk 5]
```diff
@@ -156,6 +184,9 @@ DebugClient::check_region_consistency: RpcFailure(RpcStatus { status: Unknown, d
 > - This command only supports the remote mode.
 > - Even if this command returns `success!`, you need to check whether TiKV panics. This is because this command is only a proposal that requests a consistency check for the leader, and you cannot know from the client whether the whole check process is successful or not.
 
+### Dump snapshot meta
```
Comment (lilin90) [inline tools/tikv-control.md]: Please add a blank line after this line.

**Comment by lilin90** [comment]
@hicqu Please sign the CLA.

**Comment by hicqu** [comment]
@nolouch PTAL thanks.

**Pull request merged**